### DISCUSSION TITLE
Syncing Issue[BUG]

### DISCUSSION THREAD

**Discussion by Jaydenkv** [Chia-Network/chia-blockchain#3486] [Support]
**Syncing Issue, I am not sure whether this is bug or lot. I has been noticed from Full Node Status. My status keep changing from Not Synced to Syncing and to Not Synced. Sometime it's showed Syncing, but sometime it shows Not Synced. However, the connection status remains in Connected. Is that something that I need to worry or pending to fix? Secondly, I am noticed my wallet status always show Not Synced, is that normal? Please advise, thanks. **

- OS: Windows
- OS Windows 10 Pro
- CPU: AMD Ryzen 5 Pro 3400G

**Comment by Shnurockvova** [comment]
+1

**Comment by elforin22** [comment]
me pasa lo mismo y de la nada nunca mas conecto alguna solucion?

**Reply by siliciuss** [reply]
Fellow friend. did you use google DNSs?

**Comment by elforin22** [comment]
en este foro nadie atiende los problemas?

**Reply by siliciuss** [reply]
Fellow friend, everyone is busy with it, so, please use the Google DNSs, you will see better results.

**Comment by sargonas** [comment]
While  not a one size fits all solution to all syncing problems, I would first make sure that your router has port 8444 open and forwarding to your node computers IP address. That usually goes a long way towards speeding up and resolving syncing issues, though admittedly not every one of them

**Comment by BrandtH22** [comment]
@Jaydenkv , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
